games-emulation/mupen64plus-core/mupen64plus-core-2.5.9-r4.ebuild

```diff
@@ -53,6 +53,9 @@ src_prepare() {
 	sed -i -e 's:-mmmx::g' -e 's:-msse::g' projects/unix/Makefile || die
 	# fix building against opencv-4
 	sed -i -e '/PKG_CONFIG/s:opencv:&4:' projects/unix/Makefile || die
+	# fix building against >=libbfd-2.39
+	has_version '>=sys-libs/binutils-libs-2.39' &&
```

Comment:
You're sure you can't do this another way? Could you look at the old binutils tracker on bugs.gentoo.org, and see how the other patches did it at the time? Thanks.